extensions/google-drive/src/utils.ts

```diff
@@ -148,7 +115,7 @@ const clearLeastAccessedFilePreviewsCache = (previewFiles: Array<string>) => {
   });
 };
 
-export const initialSetup = () => {
+export const initialSetup = async () => {
```

Comment:
Think there's no need to make this an async function anymore.

Reply:
Resolved in ec741b6